src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/global.scss

```diff
@@ -0,0 +1,20 @@
+@import 'normalize.css/normalize.css';
+@import './colors.scss';
+
+$bodyBackgroundColor: $wildSand;
+$h1TextColor: $blueZodiac;
+
+*, 
+*:after,
+*:before { 
```

Comment:
if we are strict we should use `::` for [pseudo element](https://developer.mozilla.org/en/docs/Web/CSS/Pseudo-elements) and `:` for [pseudo classes](developer.mozilla.org/en/docs/Web/CSS/Pseudo-classes) /cc @danrot 

possible to validate with: https://github.com/stylelint/stylelint/tree/master/lib/rules/selector-pseudo-element-colon-notation